recipe/meta.yaml

```diff
@@ -22,6 +22,8 @@ build:
   skip: true  # [win and vc<14]
   ignore_run_exports_from:
     - openssl
+  missing_dso_whitelist:
+    - '*/libomp.dylib'    # [osx]
```

Comment:
That one is worrying. I can investigate this one.

Reply:
I stumbled upon this before. I believe it is because of overlinking. If you have a good method for solving this kind of over dependency let me know. I'm afraid the only way is to dig into the upstream build system.

Reply:
From what I see, it's correctly being linked and found:
```
   INFO (sleef,lib/libsleefdft.3.5.1.dylib): Needed DSO lib/libomp.dylib found in pkgs/main::llvm-openmp-14.0.6-hc6e5704_0
```